CMake/SENSEIConfig.cmake.in

```diff
@@ -47,6 +47,7 @@ set(ENABLE_ASCENT @ENABLE_ASCENT@)
 
 include(thread)
 include(sMPI)
+include(sVTK)
```

Comment:
OK, but this should only be included if `ENABLE_VTK_CORE` is set because when all VTK features are off this file is not generated. The include probably should come after VTK and ParaView are found. It's needed for both. I tested with:
```
 67 if (ENABLE_VTK_CORE)
 68   include(sVTK)
 69 endif()
```